app/wama.web/Views/Patron/Create.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@{
```

Comment:
1st, this comment has moved (I've got no idea why). It belongs to the _**CreateNewUserAccount.cshtml**_ 

These comments, still aren't fully addressed.